### PR TITLE
[AppService] Register App Service Resources for Private Link Powershells

### PR DESCRIPTION
**Pull request by jiansong-msft**
## Description
Register App Service Resources ('Microsoft.Web/sites', 'Microsoft.Web/hostingEnvironments') for Private Link Powershells

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Network/Network/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,12 @@
 --->
 
 ## Upcoming Release
+* Register App Service Resources ('Microsoft.Web/sites', 'Microsoft.Web/hostingEnvironments') for Private Link Powershells.
```
Comment (msJinLei) [inline src/Network/Network/ChangeLog.md:22]: The private link registration is a special case where changelog description is no need.

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Review by msJinLei — CHANGES_REQUESTED** [review]
Please join https://github.com/Azure

**Comment by jiansong-msft** [comment]
Hi Jin, wondering what should I do next? Is the CI running?

**Review by msJinLei — APPROVED** [review]

**Pull request merged**